WorkingModel/Activities/Architecture_Review_Methodology.md

```diff
@@ -27,7 +27,7 @@ For each security service, the Architecture Review will identify the product com
 
 ## Workflow
 
-![Image of Architecture Review Methodology](Architecture_Review_Methodology_files/_19_0_3_43701b0_1585746146678_950815_14100.svg)
+![Image of Architecture Review Methodology](./Architecture_Review_Methodology_files/workflow.svg)
```

Comment:
@JDziurlaj I'm not sure if there was a reason for the filename, but I changed it.